Doc/library/imaplib.rst

```diff
@@ -307,6 +308,84 @@ An :class:`IMAP4` instance has the following methods:
    of the IMAP4 QUOTA extension defined in rfc2087.
 
 
+.. method:: IMAP4.idle(duration=None)
```

Comment:
I missed this one in my first round of review, but you might want to use `.. versionadded:: next` somewhere to indicate when this feature has been added (i.e. which version) (The `next` would automatically be converted into 3.14).

I don't really know whether it's better to place it before or after the example. A similar argument applies to `burst`.

Reply:
Thanks for that. If we ever need to add additional parameters to either of the two methods, we'll duplicate the `versionadded` and add a `versionchanged` so that it's similar to the rest of the docs. But for now it's perfect (and I don't think we'll ever change this interface, unless for some good reason)